Electron-Basic-Video-Chat/README.md

```diff
@@ -0,0 +1,160 @@
+overview](https://tokbox.com/opentok/tutorials/create-token/).
+
+**API key** -- The API key identifies your OpenTok developer account.
+
+## Connecting to the session
+
+First, this method initializes a Session object:
+
+    // Set Credentials
+    const apiKey = "";
+    const sessionId = "";
+    const token = "";
+
+Upon obtaining the session ID, token, and API, we initialize the session.
+
+    // Initialize Session Object
+    const session = OT.initSession(apiKey, sessionId);
+
+The `OT.initSession()` method takes two parameters -- the OpenTok API key and the session ID. It
+initializes and returns an OpenTok Session object.
+
+The `connect()` method of the Session object connects the client application to the OpenTok
+session. You must connect before sending or receiving audio-video streams in the session (or before
+interacting with the session in any way). The `connect()` method takes two parameters -- a token
+and a completion handler function:
+
+    // Connect to the Session
+    session.connect(token, (error) => {
+      
+      // If the connection is successful, initialize a publisher and publish to the session
+      if (error) {
+        return console.log('error connecting to session', error);
+      } else {
+        // Create a publisher
+        // Publish
+      }
+    });
+
+
+An error object is passed into the completion handler of the `Session.connect()` method if the
+client fails to connect to the OpenTok session. Otherwise, no error object is passed in, indicating
+that the client connected successfully to the session.
+
+The Session object dispatches a `streamDestroyed` event when the stream is Destroyed. The application defines an event handler for this event:
+
+session.on({
+  streamCreated: (event) => {
+    session.subscribe(event.stream, 'subscriber', (error) => {
+      if (error) console.log(`There was an issue subscribing to the stream ${event}`);
+    });
+  }
+});
+
+## Publishing an audio video stream to the session
+
+Upon successfully connecting to the OpenTok session (see the previous section), the application
+initializes an OpenTok Publisher object and publishes an audio-video stream to the session. This is
+done inside the completion handler for the connect() method, since you should only publish to the
+session once you are connected to it.
+
+The Publisher object is initialized as shown below. The `OT.initPublisher()` method takes three
+optional parameters:
+
+* The target DOM element or DOM element ID for placement of the publisher video
+* The properties of the publisher
+* The completion handler
+
+```
+const publisher = OT.initPublisher('publisher', (error) => {
+  if (error) {
+    return console.log('There was an error initializing the publisher', error);
+  }
+});
+
+```
+
+Once the Publisher object is initialized, we publish to the session using the `publish()`
+method of the Session object:
+
+session.publish(publisher, (error) => {
```

Comment:
You should also escape this into a code block.

Reply:
fixed in the new PR